hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapContainer.java

```diff
@@ -229,7 +229,6 @@ public ObjectNamespace getObjectNamespace() {
     public MultiMapValue merge(MergingEntryHolder<Data, MultiMapMergeContainer> mergingEntry, SplitBrainMergePolicy mergePolicy) {
         SerializationService serializationService = nodeEngine.getSerializationService();
         serializationService.getManagedContext().initialize(mergePolicy);
-        mergingEntry.setSerializationService(serializationService);
```

Comment:
Do you need to initialize the `mergingEntry` here as well?

Reply:
Good catch, fixed!